3-semantic-html/basic.html

```diff
@@ -1,11 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Basic semantic in HTML</title>
+    </head>
     
-</body>
+    <body>
+        <header>
+            <nav></nav>
+        </header>
+
+        <main>
+            <article>
+                <header>
+                    <h1>VIM editor</h1>
+                </header>
+                <section>
+                    <p>
+                        Vim is a clon with additions of "VI" the editor made for Unix OS, this was released to public in 1991. Vim is designed for use both from a CLI(<mark>command line interface</mark>) as and a standalone application in a graphic user interface.
+                        <blockquote>
+                            Vim's interface is not based on menus or icons but on commands given in a text user interface.
+                        </blockquote>
+                        Vim has 12 modes, here i mention the basic ones:
+                        <ul>
+                            <li>Normal</li>
+                            <li>Visual</li>
+                            <li>Select</li>
+                            <li>Insert</li>
+                            <li>Command Line</li>
+                            <li>Ex</li>
+                        </ul>
+                    </p>
```

Comment:
You can close the `<p>` tag before opening `<ul>`.  Because the next element that you have it's the list. You don't need to nest it.